src/avfoundation.cs

```diff
@@ -2573,6 +2573,9 @@ interface AVAsset : NSCopying {
 		NSString WasDefragmentedNotification { get; }
 #endif
 
+		[iOS (10, 2), Mac (10,12,2), TV (10, 2)]
```

Comment:
I hate to be that guy but I think you want TV (10,1) look at the Beta versions listed on https://developer.apple.com/download/